### PR TITLE
Feature/wake word file removal

### PR DESCRIPTION
**Pull request by chrisveilleux**
## Description
Implement code to automate the process of removing wake word files when an account is deleted.

## How to test
Delete an account with wake word samples in the Selene UI.  Then run the script to delete the associated files.

## Contributor license agreement signed?
CLA [Y]

### CODE REVIEW

File: api/account/tests/features/steps/profile.py [hunk 1]
```diff
@@ -65,6 +65,11 @@ def add_membership_to_account(context):
     context.refresh_token = generate_refresh_token(context)
     set_refresh_token_cookie(context)
     _add_membership_via_api(context)
+    acct_repository = AccountRepository(context.db)
+    membership = acct_repository.get_active_account_membership(
+        context.accounts["foo"].id
```
Comment (ken-mycroft) [inline api/account/tests/features/steps/profile.py:70]: foo? :-)

File: shared/selene/api/endpoints/account.py [hunk 2]
```diff
@@ -373,11 +374,28 @@ def _update_open_dataset_agreement(self, opt_in: bool):
...
+        """Process a HTTP DELETE request for an account."""
         self._authenticate()
         self.account_repository.remove(self.account)
         self.account_activity_repository.increment_accounts_deleted()
-        membership = self.account.membership
-        if membership is not None:
-            cancel_stripe_subscription(membership.payment_id)
+        if self.account.membership is not None:
+            cancel_stripe_subscription(self.account.membership.payment_id)
+        self._change_wake_word_file_status()
```
Comment (ken-mycroft) [inline shared/selene/api/endpoints/account.py:383]: so status is the same for members and non menmbers?

**Review by ken-mycroft — APPROVED** [review]

File: api/account/tests/features/steps/profile.py (hunk 1 above)
Comment (chrisveilleux) [reply api/account/tests/features/steps/profile.py:70]: I use "foo" and "bar" and "foobar" a lot in tests for funsies.

File: shared/selene/api/endpoints/account.py (hunk 2 above)
Comment (chrisveilleux) [reply shared/selene/api/endpoints/account.py:419]: membership is independent of opting in or out of the data agreement

**Review by ken-mycroft — COMMENTED** [review]
good to go

**Pull request merged**